package.json

```diff
@@ -16,10 +16,18 @@
     "test:e2e": "echo \"No tests implemented\""
   },
   "dependencies": {
+    "@newfold-labs/wp-module-ai": "file:../wp-module-ai",
```

Comment:
@aulisius @amartya-dev I'd prefer we didn't have this JS dependency here. Since the JS is just a simple file to setup an API call, I feel that this would be better implemented directly in any modules that need to hit that API.